Jinja autoescape (#50)
pixee/codemodder-python: src/core_codemods/docs/pixee_python_enable-jinja2-autoescape.md

> This codemod ensures you configure jinja2 to turn on autoescaping of HTML content. Unfortunately, the jinja2

Reviewer: ```suggestion
This codemod enables autoescaping of HTML content in `jinja2`. Unfortunately, the jinja2
```

Author: I'll update it but I thought (idk why) we were moving toward more active and not passive voice.

default behavior is to not autoescape when rendering templates, which makes your applications
vulnerable to Cross-Site Scripting (XSS) attacks.

> Our codemod currently checks if you forgot to turn autoescape on or if you explicitly disabled it. The change looks as follows:

Reviewer: ```suggestion
Our codemod checks if you forgot to enable autoescape or if you explicitly disabled it. The change looks as follows:
```

```diff
  from jinja2 import Environment

- env = Environment()

> - env = Environment(autoescape=False, loader=some-loader)

Reviewer: Minor thing but the `-` here and below is not valid Python syntax.

+ env = Environment(autoescape=True)
+ env = Environment(autoescape=True, loader=some-loader)
  ...
```

> At this time, this codemod will not detect if `autoescape` is assigned to a callable.

Reviewer: I understand the caveat but I think we can omit it for now.
```suggestion
```
